.gitignore

```diff
@@ -5,6 +5,8 @@
 *.settings/
 *.vs
 *.vscode
+.idea/
```

Comment:
@dgovil would you mind adding comments to this file and making the file a little more organized into categories? See this repository for some inspiration - https://github.com/github/gitignore/tree/master/Global

Reply:
Sure thing. I'll take a pass tomorrow. I assume you're fine with me moving about any of the stuff y'all already had in that case.

Reply:
Yes, please. FWIW. Apologies for some extra cleanup you will have to do here, but it will make the file easier to extend.

Reply:
Okay, organized it into 3 groups: Editor, Build and Platform. Groups denoted by `#####`. Subgroups within that denoted by `##` and then comments for the one or two ones that I felt remained ambiguous after the grouping.